Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'jsonapi-serializer', '~> 2.2' # https://rubygems.org/gems/jsonapi-serialize
 gem 'pg', '~> 1.1' # https://rubygems.org/gems/pg
 gem 'puma', '~> 5.0' # https://rubygems.org/gems/puma
 # gem 'rack-cors' # https://rubygems.org/gems/rack-cors
+gem 'acts_as_paranoid' # https://rubygems.org/gems/acts_as_paranoid
```

Comment:
Would be great to put this gem in alphabetical order into the `Gemfile`, like the other gems.

Reply:
Gemfile updated

Reply:
updated with correct gem